src/librustc_mir/build/mod.rs

```diff
@@ -141,17 +140,16 @@ impl<'a,'tcx> Builder<'a,'tcx> {
                 // to start, translate the argument patterns and collect the
                 // argument types.
                 let explicit_arg_decls =
-                    explicit_arguments
-                    .into_iter()
-                    .enumerate()
-                    .map(|(index, (ty, pattern))| {
-                        let lvalue = Lvalue::Arg(index as u32);
-                        unpack!(block = this.lvalue_into_pattern(block,
+                    explicit_arguments.into_iter()
+                                      .enumerate()
+                                      .map(|(index, (ty, pattern))| {
+                                              let lvalue = Lvalue::Arg(index as u32);
+                                              unpack!(block = this.lvalue_into_pattern(block,
```

Comment:
It's because inside a macro use. Macro uses which aren't comma separated lists of expression cannot be formatted (yet), and we take the original span.